src/components/denhaag-wrappers/paginations/Paginations.css

```diff
@@ -0,0 +1,10 @@
+.denhaag-pagination {
+  display: flex;
+  justify-content: center;
+  padding: var(--nlportal-space-inline-md);
+}
+
+.denhaag-pagination__link {
+  padding-right: var(--nlportal-space-inline-md);
+  text-decoration: none;
+}
```

Comment:
The styling of this Pagination wrapper is not the same as on the storybook. I think it has to do with this css being (much) too little.

Reply:
Tip: the best way to find the right css is by searching within their repository and to use that.